test/build/bundle_test.js

```diff
@@ -22,7 +22,7 @@ const bundle = require('../../lib/build/bundle');
 const Bundler = bundle.Bundler;
 const StreamAnalyzer = analyzer.StreamAnalyzer;
 
-const root = '/root';
+const root = path.resolve('/root');
```

Comment:
nit: can we make this variable name more descriptive? At first glance this looks like a loaded function/object